multi-watch/settingmanager.cpp

```diff
@@ -0,0 +1,46 @@
+#include "settingmanager.h"
+
+SettingManager::SettingManager()
+{
+
+}
+
+void SettingManager::saveTabOrder(QTabWidget *tabWidget)
+{
+  QSettings settings("multiWatchSetting", "multiWatch");
+  int nTab = tabWidget->count();
```

Comment:
Please avoid short naming. Make names like `numberOfTabs`. This is just example, fix if there are others like this